blog/urls.py

```diff
@@ -6,6 +6,7 @@
 urlpatterns = [
     path("", views.homepage, name="homepage"),
     path("create-post/", views.create_post, name="create_post"),
+    path("category/<str:cat>/", views.categories, name="category"),
```

Comment:
This line lets users define the category and see the posts related to the chosen category.